src/JSONPrinting.cpp

```diff
@@ -1,22 +1,18 @@
 #include "JSONPrinting.h"
 #include <algorithm>
 #include <fmt/format.h>
+#include <iostream>
```

Comment:
I can't spot where this is used, can it be removed?
If you used it for displaying a temporary debug message then note you could use fmt for that now instead, as you have included the header for it anyway:
```
fmt::print("Hello, {}!", "world");
```